### ISSUE TITLE
Put Tags and Submodules in separate tabs

### ISSUE THREAD

**Issue by JeReT** [ForkIssues/TrackerWin#1653]
In the latest version, stashes moved into an own section accessible through a mini-tab.
Now we have "Branches", "Stashes" and "Search Commits" as tabs.

However, inside "Branches" there are also Tags and Submodules. Both are not really branches and should not be in this tab.
To make it consistent and easier to find what you are looking for, there should also be a tab "Tags" and "Submodules".

PS: I personally would also like to have a separation between local branches and remote branches. But I'm not sure if others would like this separation as well.

**Comment by confusedsushi** [comment]
Personally I do not like that stashes are now in a separate mini tab. I also like how submodules, tags, local and remote branches are listed today. So if you want to change it, please make it configurable. That said also an option to show the stashes in their old location would be great.

**Comment by Stannieman** [comment]
I agree with @confusedsushi. I (can't speak for others) generally have enough vertical space to show all relevant stuff in 1 tab so I don't like that things get "hidden".

Specifically for stashes it is annoying cause I use them pretty frequently and with the separate tab I'm afraid I'll forget that I had something stashed. With the old approach it as right there staring at me.

**Comment by pixiekat** [comment]
Personally, I like them in individual tabs but +1 for options. Options are always good so I'd love to see people have the choice to use either layout.

**Comment by JeReT** [comment]
Okay, what about this: we have separate tabs for every kind of thing but also a tab that shows everything. So we would have the following tabs:

- Everything
- Local Branches
- Remote Branches
- Tags
- Stashes
- Submodules
- Search Commits

Fork should then remember what was lastly open. some might want to work with "Everything", some might like to see the "Local Branches" only...
